lnbits/core/tasks.py

```diff
@@ -53,6 +54,11 @@ async def wait_for_paid_invoices(invoice_paid_queue: asyncio.Queue):
                 except (httpx.ConnectError, httpx.RequestError):
                     pass
 
+        wallet = await get_wallet(payment.wallet_id)
+        await websocketUpdater(
+            wallet.inkey, {"balance": wallet.balance, "payment": payment.dict()}
+        )
```

Comment:
mypy requires you to handle the case if `wallet == None` here.

```suggestion
        if wallet:
                await websocketUpdater(
                    wallet.inkey, {"balance": wallet.balance, "payment": payment.dict()}
                )
        else:
                ...
```